rambo/vagrant_resources/vagrant/core

```diff
@@ -52,3 +52,51 @@ Vagrant.configure("2") do |config|
     puts "Vagrant done with destroy."
   end
 end
+
```

Comment:
I'd also like this entire block (all your additions) to be put in front of the block of code running `vagrant destroy` ~ L48, since it's conceptually still about setting up the VM.